File: exercicios/1180/1180.py
n=int(input())

a = input()

a = a.split()

for i in range(len(a)):
    a[i] = int(a[i])

    menor = a[0]
    posicao = 0
    for j in range(1,len(a)):
        if a[j] < menor:
            print('1')        
        if a[j] >= 10 and a[j] <= 20:
            print('2')
        if a[j] >= 20:
            print('3')

File: exercicios/JavaScript/1180/1180.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var n =  parseInt(lines.shift());
var line1 = lines.shift().split(' ');
var list = [n];
var aList = [n];
for(let i=0; i<n; i++){
    var valor = line1[i];
    list[i] = valor;
    aList[i] = valor;
}

var nList = list.sort();

console.log(`Menor valor: ${nList[0]}`);
console.log(`Posicao: ${aList.indexOf(nList[0])}`);